lib/prefix.c

```diff
@@ -236,6 +236,8 @@ afi2str(afi_t afi)
 	return "IPv6";
     case AFI_ETHER:
 	return "ethernet";
+    case AFI_L2VPN:
```

Comment:
I see in some of the earlier commits we have different case for these strings.  Is this important to get them consistent?